test/components/views/messages/MessageActionBar-test.tsx

```diff
@@ -468,4 +471,33 @@ describe("<MessageActionBar />", () => {
         fireEvent.contextMenu(queryByLabelText("Options")!);
         expect(queryByTestId("mx_MessageContextMenu")).toBeTruthy();
     });
+
+    describe("pin button", () => {
+        beforeEach(() => {
+            // enable pin button
+            jest.spyOn(SettingsStore, "getValue").mockReturnValue(true);
+        });
+
+        afterEach(() => {
+            jest.spyOn(
+                room.getLiveTimeline().getState(EventTimeline.FORWARDS)!,
+                "mayClientSendStateEvent",
+            ).mockRestore();
+        });
+
+        it("should not renders pin button when user can send state event", () => {
```

Comment:
```suggestion
        it("should not render pin button when user can't send state event", () => {
```

I this this was the wrong way around?